server/dash/dashboards/overviewComb.py

```diff
@@ -0,0 +1,199 @@
+import textwrap
+import datetime
+import urllib.parse
+import pandas as pd
+import plotly.express as px
+import plotly.graph_objects as go
+from dash import dcc, html
+
+from config import API_HOST
+from design import LABELS
+
+TITLE = "OVERVIEW COMBINED DASHBOARD"
+REPORT_API_PATH_ROOT = "/reports?"
+# DATA
+
+# Loading the dataframe for the NCs and correspoding requests
+nc_req_count_api_params = {'field':'council_name', 'filter':'created_date>=2016-01-01'}
```

Comment:
We should make a helper function that takes a `params` dict and returns the resulting df.

Reply:
Thanks for the suggestion - resolved.